packages/patternfly-4/react-table/src/components/Table/utils/decorators/classNames.js

```diff
@@ -0,0 +1,21 @@
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly/components/Table/table.css';
+
+const pickProperties = (object, properties) =>
+  properties.reduce((picked, property) => ({ ...picked, [property]: object[property] }), {});
```

Comment:
Probably a better way to do (and standardize) this across more components than just table, but this works for now.

Reply:
Yeah I agree, I don't think there is a clean way to get a subset of an object's properties. I figured this was nicer than the repetition of my original version:

```
const {
  hidden,
  hiddenOnSm,
  hiddenOnMd,
  hiddenOnLg,
  hiddenOnXl,
  visibleOnSm,
  visibleOnMd,
  visibleOnLg,
  visibleOnXl
} = styles.modifiers;

export const Visibility = {
  hidden,
  hiddenOnSm,
  hiddenOnMd,
  hiddenOnLg,
  hiddenOnXl,
  visibleOnSm,
  visibleOnMd,
  visibleOnLg,
  visibleOnXl
};
```